tsconfig.json

```diff
@@ -5,8 +5,9 @@
 		"module": "commonjs",
 		"noErrorTruncation": true,
 		"noFallthroughCasesInSwitch": true,
-		"noUnusedLocals": true,
+		"noImplicitAny": true,
```

Comment:
I think this might be implied by `strict=true`?

Reply:
I can't find a reference to it under `--strict` [here](https://www.typescriptlang.org/docs/handbook/compiler-options.html) @wrboyce